pkg/debugapi/balances.go

```diff
@@ -22,8 +23,8 @@ var (
 )
 
 type balanceResponse struct {
-	Peer    string `json:"peer"`
-	Balance int64  `json:"balance"`
+	Peer    string   `json:"peer"`
+	Balance *big.Int `json:"balance"`
```

Comment:
openapi spec should probably be modified, since this is a special data type and should be treated as a string natively in js i believe

Reply:
in the json itself it is an integer (apparently integers in json don't have an upper bound). but I guess this might make problem with `JSON.parse` @paxthemax opened this issue back when we introduced `big.Int`s in debug api: https://github.com/ethersphere/bee/issues/756.